src/test/java/uk/nhs/prm/repo/ehrtransferservice/utils/TestDataUtility.java

```diff
@@ -10,16 +10,15 @@ private TestDataUtility() { }
     public static final String NHS_NUMBER = "9798547485";
     public static final String SOURCE_GP = "B45744";
     public static final String NEMS_MESSAGE_ID = "2d74a113-1076-4c63-91bc-e50d232b6a79";
-    public static final String DESTINATION_GP = "A74854";
     public static final String NEMS_EVENT_LAST_UPDATED = "2023-10-09T15:38:03.291499328Z";
+    public static final String REPOSITORY_ODS_CODE = "REPOSITORY_ODS_CODE";
     public static final String REPOSITORY_ASID = "REPOSITORY";
```

Comment:
Why is this not `REPOSITORY_ASID = "REPOSITORY_ASID"`?

Reply:
Changed.